lib/hourglass.rb

```diff
@@ -1,4 +1,21 @@
 
 def hourglass(matrix)
```

Comment:
👍 Well done, I do suggest using row and col as variable names instead of i and j.